Source/Krypton Toolkit/Examples/Scrollbars/Form1.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Windows.Forms;
+
+namespace Scrollbars
+{
+    public partial class Form1 : Form
```

Comment:
`Krypton Form` to show themeing.

Reply:
It is not possible to do this without importing `Krypton.Toolkit`, as for some reason none of the controls in the extended toolkit show up in the toolbox, even though there are no build errors.

![image](https://user-images.githubusercontent.com/949607/115515943-89b6c380-a27d-11eb-9bd4-5b555062958c.png)

Reply:
Found the issue, I needed to temporarily put the project into `net48` for anything to work